_translator-files/po/de/Custom-Directories.po

```diff
@@ -3,93 +3,94 @@
 
 #. type: Plain text
 #: ../wiki/en/Custom-Directories.md:18
 msgid "To run a Server as a Directory, it should be started with the `--directoryserver` option to make itself (that is, 'localhost' or '127.0.0.1') the Directory to query for Servers."
-msgstr ""
+msgstr "Um einen Server als Verzeichnis zu betreiben, sollte er mit der Option `--directoryserver` gestartet werden, um sich selbst (d.h. 'localhost' oder '127.0.0.1') zum Verzeichnis für die Abfrage nach Servern zu machen."
 
 #. type: Plain text
 #: ../wiki/en/Custom-Directories.md:20
 msgid "When running a Server with the GUI, set the Custom Directory server address in the Options tab to \"localhost\", then select \"Custom\" from the Directory drop-down list."
-msgstr ""
+msgstr "Wenn du einen Server mit der grafischen Benutzeroberfläche betreibst, setze die Adresse des benutzerdefinierten Verzeichnisservers auf der Registerkarte Optionen auf \"localhost\" und wähle dann \"Benutzerdefiniert\" aus der Dropdown-Liste Verzeichnis."
 
 #. type: Plain text
 #: ../wiki/en/Custom-Directories.md:23
 msgid "### Points to note about Directories"
-msgstr ""
+msgstr "### Wissenswertes über Verzeichnisse"
 
 #. type: Plain text
 #: ../wiki/en/Custom-Directories.md:25
 msgid "If you want to control which Servers can register with your Directory, you can enable a whitelist with the `--listfilter` command line option in the format `ip address 1[;ip address 2]`."
-msgstr ""
+msgstr "Wenn du kontrollieren möchtest, welche Server sich in deinem Verzeichnis registrieren können, kannst du mit der Befehlszeilen Option `--listfilter` eine Whitelist im Format `ip address 1[;ip address 2]` aktivieren."
 
 #. type: Plain text
 #: ../wiki/en/Custom-Directories.md:27
 msgid "When running a Directory behind a NAT firewall on a private network, use the `--serverpublicip` option to specify the public IP address of the Server(s) being listed by your Directory if those Servers are on the same LAN / with the same public IP as the Directory. This is necessary to allow Clients on the public Internet to connect to them via NAT. Note that for the Servers using this option, you will still need proper port forwarding in your router/firewall."
-msgstr ""
+msgstr "Wenn du ein Verzeichnis hinter einer NAT-Firewall in einem privaten Netzwerk betreibst, verwende die Option `--serverpublicip`, um die öffentliche IP-Adresse des/der Server(s) anzugeben, der/die von deinem Verzeichnis aufgelistet wird/werden, wenn sich diese Server im gleichen LAN / mit der gleichen öffentlichen IP wie das Verzeichnis befinden. Dies ist notwendig, damit sich Clients im öffentlichen Internet über NAT mit deinem Server verbinden können. Beachte, dass du für die Server, die diese Option verwenden, trotzdem eine korrekte Portweiterleitung in deinem Router/Firewall einrichten musst."
```

Comment:
@pljones I think NAT != firewall? Probably worth changing the english translation too.

Reply:
Ok to wait for english translation?